### PR TITLE
Abranch1: base is develop

### PR DESCRIPTION
**Pull request by gudmdharalds**

### CODE REVIEW

**Comment by gudmdharalds** [comment]
test jebb test

**Comment by gudmdharalds** [comment]
test jebb test

File: bla-4.php [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<?php
```
Comment (gudmdharalds-a8c) [inline bla-4.php]: :information_source: **Info**: File is approved in review database (hashes-to-hashes) (*WordPressVIPMinimum.Info.ApprovedHashesToHashesAPI*).

File: bla-5.php [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+echo esc_html('nei');
+echo esc_html('nei');
+echo esc_html('nei');
+echo esc_html('nei');
+echo esc_html('nei');
+echo esc_html('nei');
+echo esc_html( 'nei' );
+
+echo time() . 'foo1' .  time() . 'foo2' . PHP_EOL;
+echo time() . 'foo1' .  time() . 'foo2' . PHP_EOL;
```
Comment (gudmdharalds-a8c) [inline bla-5.php]: :no_entry_sign: **Error**: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'time' (*WordPress.Security.EscapeOutput.OutputNotEscaped*).

File: bla-5.php [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+echo time() . 'foo1' .  time() . 'foo2' . PHP_EOL;
```
Comment (gudmdharalds-a8c) [inline bla-5.php]: :no_entry_sign: **Error**: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'time' (*WordPress.Security.EscapeOutput.OutputNotEscaped*).

File: bla-5.php [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+echo time() . 'foo500' .  time() . 'foo500' . PHP_EOL;
```
Comment (gudmdharalds-a8c) [inline bla-5.php]: :no_entry_sign: **Error**: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'time' (*WordPress.Security.EscapeOutput.OutputNotEscaped*).

File: bla-5.php [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+echo time() . 'foo501' .  time() . 'foo501' . PHP_EOL;
```
Comment (gudmdharalds-a8c) [inline bla-5.php]: :no_entry_sign: **Error**: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'time' (*WordPress.Security.EscapeOutput.OutputNotEscaped*).

**Comment by gudmdharalds-a8c** [comment]
This Pull Request has been added to the VIP review queue and will be reviewed and updated by a member of the VIP code review team (as per your SLA). In the meantime, please review and respond to any automated feedback generated by the @wpcomvip-vipgoci-bot. (commit-ID: be63b7d2a780f78448e43bf44457ec84aa9eb831).

**Comment by gudmdharalds-a8c** [comment]
This Pull Request has been added to the VIP review queue and will be reviewed and updated by a member of the VIP code review team (as per your SLA). In the meantime, please review and respond to any automated feedback generated by the @wpcomvip-vipgoci-bot.<br><br> If you’d like to request an urgent VIP review, please add the VIP urgent review label or reply to this PR with a comment containing the word <code>urgent</code> to notify the entire VIP code review team.

**Comment by gudmdharalds-a8c** [comment]
**PHP Syntax Errors Found**


Scan performed on the code at commit 3047c287a453d7f16fe35f0875d4ffde94375f94 ([view code](https://github.com/gudmdharalds-a8c/testing123/tree/3047c287a453d7f16fe35f0875d4ffde94375f94)).

***

This bot provides automated PHP Linting and PHPCS scanning, read more [here](https://vip.wordpress.com/documentation/the-vip-code-analysis-bot/).

***

**Error**: syntax error, unexpected end of file, expecting `,` or `;`


https://github.com/gudmdharalds-a8c/testing123/blob/3047c287a453d7f16fe35f0875d4ffde94375f94/Tests/Fixtures/xml/myfile.php#L5

**Comment by gudmdharalds-a8c** [comment]
**PHP Syntax Errors Found**


Scan performed on the code at commit 3047c287a453d7f16fe35f0875d4ffde94375f94 ([view code](https://github.com/gudmdharalds-a8c/testing123/tree/3047c287a453d7f16fe35f0875d4ffde94375f94)).

***

This bot provides automated PHP Linting and PHPCS scanning, read more [here](https://vip.wordpress.com/documentation/the-vip-code-analysis-bot/).

***

**Error**: syntax error, unexpected end of file, expecting `,` or `;`


https://github.com/gudmdharalds-a8c/testing123/blob/3047c287a453d7f16fe35f0875d4ffde94375f94/Tests/Fixtures/xml/myfile.php#L5

**Comment by gudmdharalds-a8c** [comment]
**PHP Syntax Errors Found**


Scan performed on the code at commit 3047c287a453d7f16fe35f0875d4ffde94375f94 ([view code](https://github.com/gudmdharalds-a8c/testing123/tree/3047c287a453d7f16fe35f0875d4ffde94375f94)).

***

This bot provides automated PHP Linting and PHPCS scanning, read more [here](https://vip.wordpress.com/documentation/the-vip-code-analysis-bot/).

***

**Error**: syntax error, unexpected end of file, expecting `,` or `;`


https://github.com/gudmdharalds-a8c/testing123/blob/3047c287a453d7f16fe35f0875d4ffde94375f94/Tests/Fixtures/xml/myfile.php#L5

**Comment by gudmdharalds-a8c** [comment]
**PHP Syntax Errors Found**


Scan performed on the code at commit 3047c287a453d7f16fe35f0875d4ffde94375f94 ([view code](https://github.com/gudmdharalds-a8c/testing123/tree/3047c287a453d7f16fe35f0875d4ffde94375f94)).

***

This bot provides automated PHP Linting and PHPCS scanning, read more [here](https://vip.wordpress.com/documentation/the-vip-code-analysis-bot/).

***

**Error**: syntax error, unexpected end of file, expecting `,` or `;`


https://github.com/gudmdharalds-a8c/testing123/blob/3047c287a453d7f16fe35f0875d4ffde94375f94/Tests/Fixtures/xml/myfile.php#L5

**Comment by gudmdharalds-a8c** [comment]
**PHP Syntax Errors Found**


Scan performed on the code at commit 3047c287a453d7f16fe35f0875d4ffde94375f94 ([view code](https://github.com/gudmdharalds-a8c/testing123/tree/3047c287a453d7f16fe35f0875d4ffde94375f94)).

***

This bot provides automated PHP Linting and PHPCS scanning, read more [here](https://vip.wordpress.com/documentation/the-vip-code-analysis-bot/).

***

**Error**: syntax error, unexpected end of file, expecting `,` or `;`


https://github.com/gudmdharalds-a8c/testing123/blob/3047c287a453d7f16fe35f0875d4ffde94375f94/Tests/Fixtures/xml/myfile.php#L5

**Comment by gudmdharalds-a8c** [comment]
**PHP Syntax Errors Found**


Scan performed on the code at commit 3047c287a453d7f16fe35f0875d4ffde94375f94 ([view code](https://github.com/gudmdharalds-a8c/testing123/tree/3047c287a453d7f16fe35f0875d4ffde94375f94)).

***

This bot provides automated PHP Linting and PHPCS scanning, read more [here](https://vip.wordpress.com/documentation/the-vip-code-analysis-bot/).

***

**Error**: syntax error, unexpected end of file, expecting `,` or `;`


https://github.com/gudmdharalds-a8c/testing123/blob/3047c287a453d7f16fe35f0875d4ffde94375f94/Tests/Fixtures/xml/myfile.php#L5

**Comment by gudmdharalds-a8c** [comment]
**PHP Syntax Errors Found**


Scan performed on the code at commit 1aa980def422345aa1a193173da01be814e43a48 ([view code](https://github.com/gudmdharalds-a8c/testing123/tree/1aa980def422345aa1a193173da01be814e43a48)).

***

This bot provides automated PHP Linting and PHPCS scanning, read more [here](https://vip.wordpress.com/documentation/the-vip-code-analysis-bot/).

***

**Error**: syntax error, unexpected end of file, expecting `,` or `;`


https://github.com/gudmdharalds-a8c/testing123/blob/1aa980def422345aa1a193173da01be814e43a48/Tests/myotherfile.php#L5

**Comment by gudmdharalds-a8c** [comment]
**PHP Syntax Errors Found**


Scan performed on the code at commit 078cafe05d1997a848b37835263963bf15ebea60 ([view code](https://github.com/gudmdharalds-a8c/testing123/tree/078cafe05d1997a848b37835263963bf15ebea60)).

***

This bot provides automated PHP Linting and PHPCS scanning, read more [here](https://vip.wordpress.com/documentation/the-vip-code-analysis-bot/).

***

**Error**: syntax error, unexpected end of file, expecting `,` or `;`


https://github.com/gudmdharalds-a8c/testing123/blob/078cafe05d1997a848b37835263963bf15ebea60/Tests/Fixtures/xml/myfile.php#L5

File: folder2/myfile.php [hunk 6]
```diff
@@ -0,0 +1,3 @@
+<?php
```
Comment (gudmdharalds-a8c) [inline folder2/myfile.php]: :information_source: **Info**: File is approved in review database (hashes-to-hashes) (*WordPressVIPMinimum.Info.ApprovedHashesToHashesAPI*).

**Comment by gudmdharalds-a8c** [comment]
**PHP Syntax Errors Found**


Scan performed on the code at commit 078cafe05d1997a848b37835263963bf15ebea60 ([view code](https://github.com/gudmdharalds-a8c/testing123/tree/078cafe05d1997a848b37835263963bf15ebea60)).

***

This bot provides automated PHP Linting and PHPCS scanning, read more [here](https://vip.wordpress.com/documentation/the-vip-code-analysis-bot/).

***

**Error**: syntax error, unexpected end of file, expecting `,` or `;`


https://github.com/gudmdharalds-a8c/testing123/blob/078cafe05d1997a848b37835263963bf15ebea60/Tests/Fixtures/xml/myfile.php#L5

**Comment by gudmdharalds-a8c** [comment]
**PHP Syntax Errors Found**


Scan performed on the code at commit 078cafe05d1997a848b37835263963bf15ebea60 ([view code](https://github.com/gudmdharalds-a8c/testing123/tree/078cafe05d1997a848b37835263963bf15ebea60)).

***

This bot provides automated PHP Linting and PHPCS scanning, read more [here](https://vip.wordpress.com/documentation/the-vip-code-analysis-bot/).

***

**Error**: syntax error, unexpected end of file, expecting `,` or `;`


https://github.com/gudmdharalds-a8c/testing123/blob/078cafe05d1997a848b37835263963bf15ebea60/Tests/Fixtures/xml/myfile.php#L5

**Comment by gudmdharalds-a8c** [comment]
**PHP Syntax Errors Found**


Scan performed on the code at commit 078cafe05d1997a848b37835263963bf15ebea60 ([view code](https://github.com/gudmdharalds-a8c/testing123/tree/078cafe05d1997a848b37835263963bf15ebea60)).

***

This bot provides automated PHP Linting and PHPCS scanning, read more [here](https://vip.wordpress.com/documentation/the-vip-code-analysis-bot/).

***

**Error**: syntax error, unexpected end of file, expecting `,` or `;`


https://github.com/gudmdharalds-a8c/testing123/blob/078cafe05d1997a848b37835263963bf15ebea60/Tests/Fixtures/xml/myfile.php#L5

**Comment by gudmdharalds-a8c** [comment]
**PHP Syntax Errors Found**


Scan performed on the code at commit 078cafe05d1997a848b37835263963bf15ebea60 ([view code](https://github.com/gudmdharalds-a8c/testing123/tree/078cafe05d1997a848b37835263963bf15ebea60)).

***

This bot provides automated PHP Linting and PHPCS scanning, read more [here](https://vip.wordpress.com/documentation/the-vip-code-analysis-bot/).

***

**Error**: syntax error, unexpected end of file, expecting `,` or `;`


https://github.com/gudmdharalds-a8c/testing123/blob/078cafe05d1997a848b37835263963bf15ebea60/Tests/Fixtures/xml/myfile.php#L5

**Comment by gudmdharalds-a8c** [comment]
**PHP Syntax Errors Found**


Scan performed on the code at commit dc4eee6747e6b068225d9d04b2105f68eb968798 ([view code](https://github.com/gudmdharalds-a8c/testing123/tree/dc4eee6747e6b068225d9d04b2105f68eb968798)).

***

This bot provides automated PHP Linting and PHPCS scanning, read more [here](https://vip.wordpress.com/documentation/the-vip-code-analysis-bot/).

***

**Error**: syntax error, unexpected end of file, expecting `,` or `;`


https://github.com/gudmdharalds-a8c/testing123/blob/dc4eee6747e6b068225d9d04b2105f68eb968798/Tests/Fixtures/xml/myfile.php#L5

File: folder2/myfile.php [hunk 7]
```diff
@@ -0,0 +1,7 @@
...
+
+echo time() . PHP_EOL;
+echo time() . PHP_EOL;
+echo time() . PHP_EOL;
+echo time() . PHP_EOL;
```
Comment (gudmdharalds-a8c) [inline folder2/myfile.php]: :no_entry_sign: **Error**: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'time' (*WordPress.Security.EscapeOutput.OutputNotEscaped*).

File: folder2/myfile.php [hunk 8]
```diff
@@ -0,0 +1,7 @@
+<?php
+
+echo time() . PHP_EOL;
+echo time() . PHP_EOL;
+echo time() . PHP_EOL;
```
Comment (gudmdharalds-a8c) [inline folder2/myfile.php]: :no_entry_sign: **Error**: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'time' (*WordPress.Security.EscapeOutput.OutputNotEscaped*).

File: folder2/myfile.php [hunk 9]
```diff
@@ -0,0 +1,7 @@
+<?php
+
+echo time() . PHP_EOL;
+echo time() . PHP_EOL;
```
Comment (gudmdharalds-a8c) [inline folder2/myfile.php]: :no_entry_sign: **Error**: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'time' (*WordPress.Security.EscapeOutput.OutputNotEscaped*).

File: folder2/myfile.php [hunk 10]
```diff
@@ -0,0 +1,7 @@
+<?php
+
+echo time() . PHP_EOL;
```
Comment (gudmdharalds-a8c) [inline folder2/myfile.php]: :no_entry_sign: **Error**: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'time' (*WordPress.Security.EscapeOutput.OutputNotEscaped*).

**Comment by gudmdharalds-a8c** [comment]
**PHP Syntax Errors Found**


Scan performed on the code at commit dc4eee6747e6b068225d9d04b2105f68eb968798 ([view code](https://github.com/gudmdharalds-a8c/testing123/tree/dc4eee6747e6b068225d9d04b2105f68eb968798)).

***

This bot provides automated PHP Linting and PHPCS scanning, read more [here](https://vip.wordpress.com/documentation/the-vip-code-analysis-bot/).

***

**Error**: syntax error, unexpected end of file, expecting `,` or `;`


https://github.com/gudmdharalds-a8c/testing123/blob/dc4eee6747e6b068225d9d04b2105f68eb968798/Tests/Fixtures/xml/myfile.php#L5

**Comment by gudmdharalds-a8c** [comment]
**PHP Syntax Errors Found**


Scan performed on the code at commit dc4eee6747e6b068225d9d04b2105f68eb968798 ([view code](https://github.com/gudmdharalds-a8c/testing123/tree/dc4eee6747e6b068225d9d04b2105f68eb968798)).

***

This bot provides automated PHP Linting and PHPCS scanning, read more [here](https://vip.wordpress.com/documentation/the-vip-code-analysis-bot/).

***

**Error**: syntax error, unexpected end of file, expecting `,` or `;`


https://github.com/gudmdharalds-a8c/testing123/blob/dc4eee6747e6b068225d9d04b2105f68eb968798/Tests/Fixtures/xml/myfile.php#L5

**Comment by gudmdharalds-a8c** [comment]
**PHP Syntax Errors Found**


Scan performed on the code at commit f71b56a4999044acde06728e0061f3e201e4fefb ([view code](https://github.com/gudmdharalds-a8c/testing123/tree/f71b56a4999044acde06728e0061f3e201e4fefb)).

***

This bot provides automated PHP Linting and PHPCS scanning, read more [here](https://vip.wordpress.com/documentation/the-vip-code-analysis-bot/).

***

**Error**: syntax error, unexpected end of file, expecting `,` or `;`


https://github.com/gudmdharalds-a8c/testing123/blob/f71b56a4999044acde06728e0061f3e201e4fefb/Tests/Fixtures/xml/myfile.php#L5

File: folder2/myfile3.php [hunk 11]
```diff
@@ -0,0 +1,7 @@
+<?php
+
+echo time() . PHP_EOL;
+echo time() . PHP_EOL;
+echo time() . PHP_EOL;
+echo time() . PHP_EOL;
```
Comment (gudmdharalds-a8c) [inline folder2/myfile3.php]: :no_entry_sign: **Error**: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'time' (*WordPress.Security.EscapeOutput.OutputNotEscaped*).

File: folder2/myfile3.php [hunk 12]
```diff
@@ -0,0 +1,7 @@
+<?php
+
+echo time() . PHP_EOL;
+echo time() . PHP_EOL;
+echo time() . PHP_EOL;
```
Comment (gudmdharalds-a8c) [inline folder2/myfile3.php]: :no_entry_sign: **Error**: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'time' (*WordPress.Security.EscapeOutput.OutputNotEscaped*).

File: folder2/myfile3.php [hunk 13]
```diff
@@ -0,0 +1,7 @@
+<?php
+
+echo time() . PHP_EOL;
+echo time() . PHP_EOL;
```
Comment (gudmdharalds-a8c) [inline folder2/myfile3.php]: :no_entry_sign: **Error**: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'time' (*WordPress.Security.EscapeOutput.OutputNotEscaped*).

File: folder2/myfile3.php [hunk 14]
```diff
@@ -0,0 +1,7 @@
+<?php
+
+echo time() . PHP_EOL;
```
Comment (gudmdharalds-a8c) [inline folder2/myfile3.php]: :no_entry_sign: **Error**: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'time' (*WordPress.Security.EscapeOutput.OutputNotEscaped*).

**Comment by gudmdharalds-a8c** [comment]
**PHP Syntax Errors Found**


Scan performed on the code at commit 0da4584a8b94384463e9554984250ebbc7043829 ([view code](https://github.com/gudmdharalds-a8c/testing123/tree/0da4584a8b94384463e9554984250ebbc7043829)).

***

This bot provides automated PHP Linting and PHPCS scanning, read more [here](https://vip.wordpress.com/documentation/the-vip-code-analysis-bot/).

***

**Error**: syntax error, unexpected end of file, expecting `,` or `;`


https://github.com/gudmdharalds-a8c/testing123/blob/0da4584a8b94384463e9554984250ebbc7043829/Tests/Fixtures/xml/myfile.php#L5

File: folder2/myfile4.php [hunk 15]
```diff
@@ -0,0 +1,7 @@
+<?php
+
+echo time() . PHP_EOL;
+echo time() . PHP_EOL;
+echo time() . PHP_EOL;
+echo time() . PHP_EOL;
```
Comment (gudmdharalds-a8c) [inline folder2/myfile4.php]: :no_entry_sign: **Error**: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'time' (*WordPress.Security.EscapeOutput.OutputNotEscaped*).

File: folder2/myfile4.php [hunk 16]
```diff
@@ -0,0 +1,7 @@
+<?php
+
+echo time() . PHP_EOL;
+echo time() . PHP_EOL;
+echo time() . PHP_EOL;
```
Comment (gudmdharalds-a8c) [inline folder2/myfile4.php]: :no_entry_sign: **Error**: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'time' (*WordPress.Security.EscapeOutput.OutputNotEscaped*).

File: folder2/myfile4.php [hunk 17]
```diff
@@ -0,0 +1,7 @@
+<?php
+
+echo time() . PHP_EOL;
+echo time() . PHP_EOL;
```
Comment (gudmdharalds-a8c) [inline folder2/myfile4.php]: :no_entry_sign: **Error**: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'time' (*WordPress.Security.EscapeOutput.OutputNotEscaped*).

File: folder2/myfile4.php [hunk 18]
```diff
@@ -0,0 +1,7 @@
+<?php
+
+echo time() . PHP_EOL;
```
Comment (gudmdharalds-a8c) [inline folder2/myfile4.php]: :no_entry_sign: **Error**: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'time' (*WordPress.Security.EscapeOutput.OutputNotEscaped*).

**Pull request closed without merging**